src/export/ValueSetExporter.ts

```diff
@@ -176,7 +184,9 @@ export class ValueSetExporter {
               this.addConceptComposeElement(composeElement, valueSet.compose.include);
             }
           } else {
-            valueSet.compose.include.push(composeElement);
+            if (composeElement.valueSet?.length != 0) {
```

Comment:
This is nitpicky, but I think it is preferred to use `!==` here because I think we do want to be checking strict equality (or inequality) here.

```suggestion
            if (composeElement.valueSet?.length !== 0) {
```